cookbooks/aws-parallelcluster-slurm/resources/munge_key_manager.rb

```diff
@@ -0,0 +1,40 @@
+resource_name :munge_key_manager
+provides :munge_key_manager
+unified_mode true
+
+property :munge_key_secret_arn, String
+
+default_action :manage
+
+action :manage do
```

Comment:
Can we use a more descriptive action e.g. `:setup_munge_key`?

Reply:
Sure!